Example.playground/Pages/Repositories search.xcplaygroundpage/Contents.swift

```diff
@@ -1,15 +1,82 @@
+/*:
+ # FTAPIKit playground
 
+ ### Build instructions
+ - This playground is meant to be used withing Xcode.
+ - Do not open this playground directly. Use FTAPIKit.xcworkspace instead.
+ - Once you have opened the workspace, build. Make sure build target platform is the same as Playground's current platform.
+ */
 import Foundation
 import FTAPIKit
 import PlaygroundSupport
 
 PlaygroundPage.current.needsIndefiniteExecution = true
+/*:
+ ## Example description
+ This example connects to the Github API using credentials provided below and searches all user's repositories for all project using Swift language.
 
+
+ ## API Adapter declaration
+
+ ### Prepare variables and configuration
+ *Fill in credentials of some active github account.*
+ */
 let username = "your_github_username"
 let password = "your_github_password"
+
+/*:
+ Authorization method for Github is described in the link below. FTAPIKit doesn't solve authorization challenges on its own and relies on external support from either networking framework provided to FTAPIKit or custom user implementation using FTAPIKit's callbacks and delegates. However, FTAPIKit does not handle any authorization challenges on its own, FTAPIKit's APIEndpoint protocol contains property _authorized_ which indicates, whether authorization challenges do apply for the endpoint.
```

Comment:
Fact, that `FTAPIKit doesn't handle authorization challenges on its own` is repeated.
```suggestion
 Authorization method for Github is described in the link below. FTAPIKit doesn't solve authorization challenges on its own and relies on external support from either networking framework provided to FTAPIKit or custom user implementation using FTAPIKit's callbacks and delegates. FTAPIKit's APIEndpoint protocol contains property _authorized_ which indicates, whether authorization challenges do apply for the endpoint.
```